src/lib.rs

```diff
@@ -59,8 +59,12 @@
 #[cfg_attr(feature = "nightly", cfg(target_has_atomic = "ptr"))]
 pub use crossbeam_utils::atomic;
 
-/// Miscellaneous utilities.
 pub mod utils {
+    //! Miscellaneous utilities.
+    //!
+    //! * [`Backoff`], for exponential backoff in spin loops.
+    //! * [`CachePadded`], for padding and aligning a value to the length of a cache line.
```

Comment:
By the way, this changed from outer doc to inner doc because it was needed to write link like `[`Backoff`](crate:: utils::Backoff)` when using outer doc.